### PR TITLE
feat: add readMsExperiment (issue #32)

### PR DESCRIPTION
**Pull request by jorainer**

### CODE REVIEW

**Comment by lgatto** [comment]
See #32 for a question about the signature and the nature of the files.

**Comment by jorainer** [comment]
I changed now parameter `files` to `spectraFiles`.

**Comment by jorainer** [comment]
@lgatto , @sgibb - just a little ping to maybe have a look at the PR if you find the time.

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # MsExperiment 1.1
 
+## MsExperiment 1.1.2
+
+- Add `readMsExperiment` function ([issue
+  #32](https://github.com/rformassspectrometry/MsExperiment/issues/32)).
```
Comment (sgibb) [inline NEWS.md]:
```suggestion
- Add `readMsExperiment` function ([issue #32](https://github.com/rformassspectrometry/MsExperiment/issues/32)).
```

**Review by sgibb — APPROVED** [review]
All good! Thanks for adding this functionality.

**Pull request merged**